src/main/java/com/enterprise/fundraisermanager/FundraiserManagerController.java

```diff
@@ -32,6 +37,11 @@ public String index(Model model){
         return "start";
     }
 
+    /**
+     * Endpoint to save Fundraiser.
+     * @param fundraiser
+     * @return start page
+     */

```

Comment:
1 example where I added Javadoc to methods

Reply:
Well written JavaDoc should describe more than what you can ascertain from reading the method signature itself.